roles/gpu/tasks/gpu.yml

```diff
@@ -0,0 +1,67 @@
+---
+- name: Install yum requirements for gpu driver installation
+  ansible.builtin.yum:
+    state: 'installed'
+    update_cache: true
+    name:
+      - tar
+      - bzip2
+      - make
+      - automake
+      - gcc
+      - gcc-c++
+      - pciutils
+      - elfutils-libelf-devel
+      - libglvnd-devel
+      - bind-utils
+      - wget
+  become: true
+
+# ansible_kernel variable is not working, as after reboot, still holds old kernel
```

Comment:
Can this be handled with a task to update a specific "gathered fact". E.g. something like this:
https://stackoverflow.com/questions/35185635/can-ansible-fetch-updated-facts-in-middle-of-a-playbook

Reply:
Is running `ansible.builtin.setup` step any better than running `uname -r`?
For uname I can at least trust that it will always work :smile:. Of course I can change it, if you think this is better in any way.

Reply:
Updated to setups :heavy_check_mark: